site/docs/spark-queries.md

```diff
@@ -191,57 +192,166 @@ join prod.db.table.snapshots s
-| path                                                                 | length | partition_spec_id | added_snapshot_id   | added_data_files_count | existing_data_files_count | deleted_data_files_count | partition_summaries                  |
-+----------------------------------------------------------------------+--------+-------------------+---------------------+------------------------+---------------------------+--------------------------+--------------------------------------+
+
+<div class="markdown-table-container" markdown="block">
+| path | length | partition_spec_id | added_snapshot_id | added_data_files_count | existing_data_files_count | deleted_data_files_count | partition_summaries |
+| -- | -- | -- | -- | -- | -- | -- | -- |
 | s3://.../table/metadata/45b5290b-ee61-4788-b324-b1e2735c0e10-m0.avro | 4479   | 0                 | 6668963634911763636 | 8                      | 0                         | 0                        | [[false,null,2019-05-13,2019-05-15]] |
-+----------------------------------------------------------------------+--------+-------------------+---------------------+------------------------+---------------------------+--------------------------+--------------------------------------+
+</div>
+
+!!! Note
+    1. Fields within `partition_summaries` column of the manifests table correspond to `field_summary` structs within [manifest list](./spec.md#manifest-lists), with the following order: 
+        - `contains_null`
+        - `contains_nan`
+        - `lower_bound`
+        - `upper_bound`
+    2. `contains_nan` could return null, which indicates that this information is not available from files' metadata. 
+        This usually occurs when reading from V1 table, where `contains_nan` is not populated.
+
+### Partitions
+
+To show a table's current partitions
+
+```sql
+SELECT * FROM prod.db.table.partitions
 ```
 
-Note: 
-1. Fields within `partition_summaries` column of the manifests table correspond to `field_summary` structs within [manifest list](./spec.md#manifest-lists), with the following order: 
-   - `contains_null`
-   - `contains_nan`
-   - `lower_bound`
-   - `upper_bound`
-2. `contains_nan` could return null, which indicates that this information is not available from files' metadata. 
-   This usually occurs when reading from V1 table, where `contains_nan` is not populated. 
+<div class="markdown-table-container" markdown="block">
+| partition | record_count | file_count |
+| -- | -- | -- |
+|  {20211001, 11}|           1|         1|
+|  {20211002, 11}|           1|         1|
+|  {20211001, 10}|           1|         1|
+|  {20211002, 10}|           1|         1|
+</div>
+
+### All Metadata
+
+!!! Note
+    The metadata is readable from any snapshot currently tracked by the table
+
+!!! WARNING
+    The table's metadata may return **duplicate** rows
```

Comment:
This sentence is misleading. It should say something like `The "all" metadata tables may produce more than one row per data file or manifest file because metadata files may be used in more than one table snapshot`